click/core.py

```diff
@@ -1642,7 +1642,8 @@ def get_default(self, ctx):
                 if param.name == self.name and param.default:
                     return param.flag_value
             return None
-        return Parameter.get_default(self, ctx)
+        else:
```

Comment:
Can we leave this change out? It's not really necessary and a matter of taste, also I'm a proponent of minimal, topical PRs